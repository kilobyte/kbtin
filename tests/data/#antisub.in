#var todo {{#anti foo} {#unanti foo} {#anti bar;#anti baz} {#unanti b*} {#end}}
#act GO {#show .;#split cur todo {$todo};#if 1 {$$cur};foo;bar;baz;quux;GO}
#run a //selfpipe
#gag {%0}
GO
