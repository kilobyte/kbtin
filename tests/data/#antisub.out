.
foo
.
.
bar
baz
.
.

#SESSION 'a' DIED.
